kubernetes/secrets/bridge-config/appservice-registration-irc.yaml.erb

```diff
@@ -0,0 +1,15 @@
+id: <%= bridge_id %>
+hs_token: <%= bridge_hs_token %>
+as_token: <%= bridge_as_token %>
+namespaces:
+  users:
+    - exclusive: true
+      regex: '@.*_irc:matrix\.ocf\.berkeley\.edu'
+  aliases:
+    - exclusive: true
+      regex: '#.*:matrix\.ocf\.berkeley\.edu'
+url: 'http://matrix-bridge-service.app-matrix.svc.cluster.local:9995'
+sender_localpart: matrix-bridge
+rate_limited: false
+protocols:
+  - irc
```

Comment:
iirc this file is generated, we may want to make sure that the current version matches the generated one

Reply:
Thought the same. In the future, perhaps it could helpful to write a helper script to build it from the bridge config.yaml.erb and re-template the result using the provided the docker image? Would be willing to branch this out into a separate issue if needed.

It's also fine if this is manually written and updated too.